akka-distributed-data/src/main/scala/akka/cluster/ddata/PNCounterMap.scala

```diff
@@ -71,17 +71,27 @@ final class PNCounterMap[A] private[akka] (
 
   def size: Int = underlying.size
 
+  /**
+   * Increment the counter by 1.
+   */
+  def increment(key: A)(implicit node: SelfUniqueAddress): PNCounterMap[A] =
```

Comment:
missing `increment(key: A, delta: Long)`

Let's skip the default param value of 1, i.e. change this one to `increment(key: A, delta: Long)`

Reply:
but I think this is in conflict with
```
def increment(key: A, delta: Long = 1)(implicit node: Cluster): PNCounterMap[A]
```

so I guess we have to remove it, or invent a new name

Reply:
Indeed, that is why I added it - as a workaround due to conflict.